src/viur/datastore/query.py

```diff
@@ -287,6 +287,20 @@ def order(self, *orderings: Tuple[str, 'SortOrder']) -> 'Query':
 		if self.queries is None:
 			# This Query is unsatisfiable - don't try to bother
 			return self
+
+		# Check for correct order subscript
+		orders = []
+		for order in orderings:
+			if isinstance(order, str):
+				order = (order, SortOrder.Ascending)
+
+			assert isinstance(order[0], str) and isinstance(order[1], SortOrder), \
+				f"Invalid ordering {order}, it has to be a tuple. Try: `(\"{order}\", SortOrder.Ascending)`"
```

Comment:
In my option it's a bad practise to use an `assert` statement for argument validation (which is already done to often wrong in many ViUR-project…).
I think to raise a `TypeError` is more suitable for this:
```suggestion
			if not (isinstance(order[0], str) and isinstance(order[1], SortOrder)):
				raise TypeError(f"Invalid ordering {order}, it has to be a tuple. Try: `(\"{order}\", SortOrder.Ascending)`")
```

From the [python docs](https://docs.python.org/3/library/exceptions.html#TypeError): 
>Passing arguments of the wrong type (e.g. passing a [list](https://docs.python.org/3/library/stdtypes.html#list) when an [int](https://docs.python.org/3/library/functions.html#int) is expected) should result in a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError), but passing arguments with the wrong value (e.g. a number outside expected boundaries) should result in a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError).

Reply:
I agree on replacing assert statements with decent Exception handling.